distribution/docs/src/main/resources/_contents/_resource-management-contents/managing-resource-management-contents.adoc

```diff
@@ -0,0 +1,48 @@
+
+The ${ddf-resource-management} Application provides administrative functionality to impose data usage limits on users, as well as the ability to view and terminate searches that are in progress.
```

Comment:
probably should add a note here that it allows an admin to view data usage as well as impose limits.  Perhaps a list of capabilities is appropriate here.. view data usage, impose data limits, view and cancel running searches...